virtual_ecosystem/models/hydrology/hydrology_model.py

```diff
@@ -135,7 +132,7 @@ def __init__(
         """Set of constants for the hydrology model"""
         self.core_constants = core_components.core_constants
         """Set of core constants for the hydrology model"""
-        self.data.grid.set_neighbours(distance=sqrt(self.data.grid.cell_area))
+        self.grid.set_neighbours(distance=sqrt(self.grid.cell_area))
```

Comment:
Was going to do this in another PR, but ahead of the curve 😄

Reply:
upsi, I hope it won't break anything